khelo/assets/locales/app_en.arb

```diff
@@ -239,6 +239,16 @@
   "tournament_detail_stats_avg_title": "Avg",
   "tournament_detail_stats_runs_title": "Runs",
 
+  "tournament_detail_points_table_points_title": "Points",
+  "tournament_detail_points_table_empty_title": "No Points Yet!",
+  "tournament_detail_points_table_empty_description": "You don't have any points yet. Join a match to start earning points!",
```

Comment:
"join match to earn point..." this doesn't make sense as point represents the result of matches of team, i.e win: 2, lose: 0, tie: 1.